app/views/work_packages/reports/report.html.erb

```diff
@@ -28,21 +28,10 @@ See doc/COPYRIGHT.rdoc for more details.
 ++#%>
 <% html_title l(:label_summary) %>
 <%= toolbar title: l(:label_summary) %>
-<div class="grid-block">
-  <div class="grid-content">
-    <% [@type_report, @priority_report, @assignee_report, @responsible_report, @author_report].each do |report| %>
-      <%= render :partial => 'report_category', :locals => { report: report} %>
-    <% end %>
-    <%= call_hook(:view_reports_issue_report_split_content_left, :project => @project) %>
-    <%
-    additional_reports = [@version_report]
-    additional_reports << @subproject_report if @project.children.any?
-    additional_reports << @category_report
-    %>
-    <% additional_reports.each do |report| %>
-      <%= render :partial => 'report_category', :locals => { report: report } %>
-      <br />
-    <% end %>
-    <%= call_hook(:view_reports_issue_report_split_content_right, :project => @project) %>
-  </div>
-</div>
+<%  reports = [@author_report, @type_report, @priority_report, @version_report, @category_report, @assignee_report, @responsible_report]
+    reports <<  @subproject_report if @project.children.any? %>
```

Comment:
I know, you just moved that code around. But wouldn't it be better, if the `ReportsController` would provide the list of reports (`@reports`?) directly, instead of providing several reports that the view has to know by name?